eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -3224,6 +3224,84 @@ public function testMoveSubtreeUpdatesPathIdentificationString(): void
         );
     }
 
+    /**
+     * Test that is_visible is set correct for children when moving a content which is hidden (location is not hidden).
+     *
+     * @covers \eZ\Publish\API\Repository\LocationService::moveSubtree
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     */
+    public function testMoveSubtreeKeepsContentHidden(): void
+    {
+        $repository = $this->getRepository();
+
+        $mediaLocationId = $this->generateId('location', 43);
+        $demoDesignLocationId = $this->generateId('location', 56);
+        /* BEGIN: Use Case */
+        // $mediaLocationId is the ID of the "Media" page location in
+        // an eZ Publish demo installation
+
+        // $demoDesignLocationId is the ID of the "Demo Design" page location in an eZ
+        // Publish demo installation
+
+        // Load the location service
+        $locationService = $repository->getLocationService();
+
+        // Load the content service
+        $contentService = $repository->getContentService();
+
+        // Load location to move
+        $locationToMove = $locationService->loadLocation($demoDesignLocationId);
+
+        // Create child below locationToMove
+        $subFolderContent = $this->publishContentWithParentLocation('SubFolder', $locationToMove->id);
+
+        // Hide source Content
+        $contentService->hideContent($locationToMove->contentInfo);
+        $locationToMove = $locationService->loadLocation($demoDesignLocationId);
+
+        // Load new parent location
+        $newParentLocation = $locationService->loadLocation($mediaLocationId);
+
+        // Move location from "Home" to "Media"
+        $locationService->moveSubtree(
+            $locationToMove,
+            $newParentLocation
+        );
+
+        // Load moved location
+        $movedLocation = $locationService->loadLocation($demoDesignLocationId);
+        /* END: Use Case */
```

Comment:
BEGIN and END are technical debt. Those were placeholder to generate documentation, no longer used.

Reply:
Removed in 6b21d3e6f35ace455c794f770d1e253d89711129